src/modules/logger/messages.h

```diff
@@ -131,28 +161,40 @@ struct ulog_message_dropout_s {
 	uint16_t duration; //in ms
 };
 
-struct ulog_message_data_header_s {
+struct ulog_message_data_s {
 	uint16_t msg_size; //size of message - ULOG_MSG_HEADER_LEN
 	uint8_t msg_type = static_cast<uint8_t>(ULogMessageType::DATA);
 
 	uint16_t msg_id;
 };
 
-struct ulog_message_info_header_s {
+/**
+ * @brief Information Message
+ *
+ * Writes the dictionary type key:value relationship of any kind of information.
+ * Example: key_value_str[] = "char[5] sys_toolchain_ver9.4.0"
+ */
+struct ulog_message_info_s {
 	uint16_t msg_size; //size of message - ULOG_MSG_HEADER_LEN
 	uint8_t msg_type = static_cast<uint8_t>(ULogMessageType::INFO);
 
-	uint8_t key_len;
-	char key[255];
+	uint8_t key_len; // Length of the 'key'
+	char key_value_str[255]; // String with the key and value information
 };
 
-struct ulog_message_info_multiple_header_s {
+/**
+ * @brief Multiple Information Message
+ *
+ * Writes the dictionary type key:value relationship of any kind of information, but
+ * for the ones which has a long value that can't be contained in a single Information Message.
+ */
+struct ulog_message_info_multiple_s {
 	uint16_t msg_size; //size of message - ULOG_MSG_HEADER_LEN
 	uint8_t msg_type = static_cast<uint8_t>(ULogMessageType::INFO_MULTIPLE);
 
-	uint8_t is_continued; ///< can be used for arrays: set to 1, if this message is part of the previous with the same key
-	uint8_t key_len;
-	char key[255];
+	uint8_t is_continued; // Can be used for arrays: set to 1, if this message is part of the previous with the same key
```

Comment:
Please stick to doxygen comments for commenting fields (for the others below as well).
```suggestion
	uint8_t is_continued; ///< Can be used for arrays: set to 1, if this message is part of the previous with the same key
```

Reply:
Done!